inst/shiny-server/server/ERTPlot.R

```diff
@@ -93,16 +93,92 @@ get_max_targets <- function(data, aggr_on, maximize){
     Fval <- ifelse(maximize, max(Fall), min(Fall))
     targets <- c(targets,Fval)
   }
+  names(targets) <- aggr_attr
   targets
 }
 
+# render_ERTPlot_aggr_plot <- reactive({
```

Comment:
What was the problem with this function, such that you need to re-implement it? ;)

Reply:
I did this to add support for the selection of custom targets by the user, which was easier when using a different function as the starting point